prime-router/src/main/kotlin/azure/BlobAccess.kt

```diff
@@ -322,6 +395,21 @@ class BlobAccess() : Logging {
             }
         }
 
+        /**
+         * Download the file associated with a RawSubmission message
+         */
+        fun downloadContent(
```

Comment:
Might be nice to bubble the error up somehow instead of throwing/logging in the method.

Reply:
I just moved this function out of QueueMessage and over to BlobAccess so I don't see how it breaks the existing functionality. But I think I agree that a exception should be thrown from the Converter code. Since that means an empty message was added to the queue which shouldn't ever happen, right?